components/Auction/Auction.module.css

```diff
@@ -1,7 +1,20 @@
-.auctionPageNavigation {
+.newAuctionBtnContainer {
   text-align: center;
 }
 
+.newAuctionBtn {
+  font-weight: 550;
+  height: 50px;
+  font-size: 1.2rem;
+  width: 14rem;
+  border: none;
+  text-align: center;
+  padding: 2px 10px;
+  border-radius: 100px;
```

Comment:
It should be 10px right? I guess you accidentally added one more zero.

Reply:
Oh yes, sorry.